tests/integration/targets/cpu_health/tasks/main.yml

```diff
@@ -5,20 +5,61 @@
 
 - name: Run CPU health check validated content operation
   block:
-    - name: Run network.cpu validated content with health_checks operation
+    # Test Case 1: Default thresholds (warning: 80, critical: 90)
+    - name: Run network.cpu with default thresholds
       ansible.builtin.include_role:
         name: network.healthchecks.cpu
       vars:
         cpu_utilization:
           details: true
-          warning_threshold: 90
+      register: default_result
+
+    - name: Assert default threshold result
+      ansible.builtin.assert:
+        that:
+          - default_result.changed == false
+          - default_result.cpu_utilization.status == "FAIL"
+          - default_result.cpu_utilization.details.cpu_utilization.status == "FAIL"
+          - default_result.cpu_utilization.details.cpu_utilization.message == "CPU utilization is above the critical threshold"
+
+    # Test Case 2: Custom thresholds (warning: 85, critical: 95)
+    - name: Run network.cpu with custom thresholds
+      ansible.builtin.include_role:
+        name: network.healthchecks.cpu
+      vars:
+        cpu_utilization:
+          details: true
+          warning_threshold: 80
           critical_threshold: 95
-      register: result
+      register: custom_result
 
-    - name: Assert result has not changed
+    - name: Assert custom threshold result
       ansible.builtin.assert:
         that:
-          - result.changed == false
+          - custom_result.changed == false
+          - custom_result.cpu_utilization.status == "FAIL"
+          - custom_result.cpu_utilization.details.cpu_utilization.status == "FAIL"
+          - custom_result.cpu_utilization.details.cpu_utilization.message == "CPU utilization is above the critical threshold"
```

Comment:
I believe you should have structure like 
custom_result.healthchecks.cpu_utilization.status